.github/workflows/cloud-runner-integrity.yml

```
@@ -0,0 +1,167 @@
+      fail-fast: false
+      matrix:
+        test:
+          - 'cloud-runner-end2end-caching'
+          - 'cloud-runner-end2end-retaining'
+          - 'cloud-runner-hooks'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          lfs: false
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)